### ISSUE TITLE
App raises RuntimeError: "A 'SQLAlchemy' instance has already been registered ..."

### ISSUE THREAD

**Issue by holzman** [ssl-hep/ServiceX#623]
I think this is because `db.init_app()` is called twice - first in `create_app()` and second in `app.py`

```
Traceback (most recent call last):
  File "/usr/local/bin/flask", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/site-packages/flask/cli.py", line 1063, in main
    cli.main()
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/flask/cli.py", line 354, in decorator
    app = __ctx.ensure_object(ScriptInfo).load_app()
  File "/usr/local/lib/python3.10/site-packages/flask/cli.py", line 308, in load_app
    app = locate_app(import_name, name)
  File "/usr/local/lib/python3.10/site-packages/flask/cli.py", line 218, in locate_app
    __import__(module_name)
  File "/home/servicex/servicex/app.py", line 36, in <module>
    db.init_app(app)
  File "/usr/local/lib/python3.10/site-packages/flask_sqlalchemy/extension.py", line 253, in init_app
    raise RuntimeError(
RuntimeError: A 'SQLAlchemy' instance has already been registered on this Flask app. Import and use that instance instead.
```

**Comment by BenGalewsky** [comment]
Looks like some flask library update finally caused this error to become fatal...

See this [stack overflow post](https://stackoverflow.com/questions/75523569/runtimeerror-a-sqlalchemy-instance-has-already-been-registered-on-this-flask)

I, too tried commenting out the duplicate and got the table creation issues. I think this requires carefully reading the latest flask docs and determining the correct way to do this initialization and migration.

**Comment by BenGalewsky** [comment]
Closed via #630
